data/tutorials/lg_08_error_handling.md

```diff
@@ -758,10 +763,10 @@ c
 a >>= fun x -> b >>= fun y -> c
 ```
 Variables `x` and `y` may appear in `c` in the three cases. The first form isn't
-very convenient, it uses a lot of parenthesis. The second one is often the
-prefered one due to its ressemblance with regular local definitions. The third
+very convenient, it uses a lot of parentheses. The second one is often the
+preferred one due to its resemblance with regular local definitions. The third
```

Comment:
```suggestion
preferred due to its resemblance with regular local definitions. The third
```

Reply:
Need to delete "the" at the end of previous line